feature/experimental/bgp/ate_tests/base_bgp_session_parameters/base_bgp_session_parameters_test.go

```diff
@@ -0,0 +1,467 @@
+// Copyright 2021 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package base_bgp_session_parameters_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/confirm"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/telemetry"
+	"github.com/openconfig/ygot/ygot"
+)
+
+// The testbed consists of ate:port1 -> dut:port1.
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+var (
+	dutAttrs = attrs.Attributes{
+		Desc:    "To ATE",
+		IPv4:    "192.0.2.1",
+		IPv4Len: 30,
+	}
+	ateAttrs = attrs.Attributes{
+		//Desc:    "To DUT",
+		Name:    "ateSrc",
+		IPv4:    "192.0.2.2",
+		IPv4Len: 30,
+	}
+	dutlo0Attrs = attrs.Attributes{
+		Desc:    "Loopback ip1 ",
+		IPv4:    "11.11.11.11",
+		IPv4Len: 32,
+	}
+	dutlo0Attrs1 = attrs.Attributes{
+		Desc:    "Loopback ip2",
+		IPv4:    "12.12.12.12",
+		IPv4Len: 32,
+	}
+)
+
+const (
+	dutAS        = 64500
+	ateAS        = 64501
+	peerGrpName  = "BGP-PEER-GROUP"
+	netInstance  = "DEFAULT"
+	loopbackIntf = "lo0"
+	holdTime0    = 0
+	holdTime100  = 100
+	holdTime135  = 135
+	ConnTime0    = 0
+	ConnTime100  = 100
+	NegHoldTime0 = 0
+)
+
+//Configure network instance
```

Comment:
Note that for consistency's sake, we always have a space between `//` and the text.